FindWord.java

```diff
@@ -0,0 +1,39 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
+public class FindWord {
+    public static void main(String[] args) throws FileNotFoundException {
+        Scanner inputScan = new Scanner(System.in);
+        
+        String searchString = inputScan.next(); 
+        String patternString = inputScan.next(); 
+        String fileString = inputScan.next(); 
+        
+        inputScan.close();
+
+    // ---------- logic ---------------
+        if (!searchString.equals("search")) {
```

Comment:
this will look if any word has the word you are looking for in it, might not be the result you are looking for.